protobuf_to_dict/convertor.py

```diff
@@ -171,7 +171,7 @@ def _get_field_mapping(pb, dict_value, strict):
             continue
         if key not in pb.DESCRIPTOR.fields_by_name:
             if strict:
-                raise KeyError("%s does not have a field called %s" % (pb, key))
+                raise KeyError("%s does not have a field called %s" % (type(pb), key))
```

Comment:
Because str/repr of pb is empty. I considered improving that but all I needed here for debugging was type, to know which object was had a problem, and didn't see any more info as needed